### PR TITLE
Dokken and Inspec testing

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: .delivery/project.toml [hunk 1]
```diff
@@ -1,9 +1 @@
-[local_phases]
-unit = "rspec spec/"
-lint = 'cookstyle --display-cop-names --extra-details'
-syntax = "foodcritic ."
-provision = "echo skipping"
-deploy = "echo skipping"
-smoke = "echo skipping"
-functional = "echo skipping"
-cleanup = "echo skipping"
+remote_file = "https://raw.githubusercontent.com/chef-cookbooks/community_cookbook_tools/master/delivery/project.toml"
```
Comment (stevendanna) [inline .delivery/project.toml]: I'd prefer to avoid a network dependency for any of the test setup.

File: Makefile [hunk 2]
```diff
@@ -1,11 +0,0 @@
-all: delivery
```
Comment (stevendanna) [inline Makefile]: I'd prefer to keep the `Makefile` and to use it in `.travis.yml` as it make it easy to run the tests locally and be sure you are running what would run in travis.

**Comment by tas50** [comment]
@stevendanna This work for you now?

**Comment by stevendanna** [comment]
Yeah, I'll merge it when I'm back from vacation.  Thanks again!

**Comment by tas50** [comment]
@stevendanna Ping on the merge

**Pull request merged**